Index: Classe_Journal.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+># Classe Journal qui hérite de Document\r\nclass Journal(Document):\r\n    def __init__(self, titre, date_parution):\r\n        super().__init__(titre)\r\n        self.date_parution = date_parution\r\n\r\n    def __str__(self):\r\n        return f\"Journal - {super().__str__()} | Paru le: {self.date_parution}\"
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Classe_Journal.py b/Classe_Journal.py
--- a/Classe_Journal.py	(revision 771ca7febc989a2f6ceddbd62baf724a49ff8f9c)
+++ b/Classe_Journal.py	(date 1764891663866)
@@ -1,4 +1,5 @@
 # Classe Journal qui hérite de Document
+import  Classe_Document as Document
 class Journal(Document):
     def __init__(self, titre, date_parution):
         super().__init__(titre)
